docs/dictionary/property/stackFiles.lcdoc

```diff
@@ -81,16 +80,15 @@ To use a <stack> as a <library>, allowing your <handler|handlers> to
 call the <stack|stack's> <handler|handlers>, use the <start using> or
 <insert script> <command>.
 
-References: stacks (function), mainStacks (function), object (glossary),
-substack (glossary), start using (glossary), call (glossary),
-property (glossary), loaded into memory (glossary), load (glossary),
-file path (glossary), command (glossary), statement (glossary),
-LiveCode (glossary), main stack (glossary), relative file path (glossary),
-folder (glossary), script (glossary), handler (glossary), long (keyword),
-line (keyword), insert script (library), library (library),
-stack (object), substacks (property), properties (property),
-mainStack (property), defaultFolder (property), name (property),
-filename of stack (property)
+References: call (glossary), command (glossary), 
+defaultFolder (property), file path (glossary), 
+filename of stack (property), folder (glossary), handler (glossary), insert script (command), library (library), line (keyword), 
+LiveCode (glossary), load (glossary), loaded into memory (glossary), long (keyword), main stack (glossary), mainStack (property), 
```

Comment:
Could you wrap these lines so that they aren't so long please? Also, I think `library (library)` should be deleted... or maybe turned into `library (glossary)`?

Reply:
@asayd Just wanted to ping you on this so we can in case you missed this comment

Reply:
@livecodeali There is no library (glossary) that I can see. But library (library) actually reads more like a glossary entry than a library entry. It might be mis-categorized.
Scratch that. Actually, when you look at .../docs/library/l/... there is a document library.lcdoc. It looks like maybe the dictionary parser misinterprets the Type as library instead of glossary. I'll file a bug report if you think it's needed.